data/src/main/java/com/linkedin/data/DataMap.java

```diff
@@ -401,7 +401,7 @@ void disableChecker()
   }
 
   // Unit test use only
-  Map<String, Object> getUnderlying()
+  protected Map<String, Object> getUnderlying()
```

Comment:
If this is no longer just for unit tests, please remove the comment.

Reply:
Actually it is still for unit tests only. I removed the protected and retained the comment.